### PR TITLE
Add network policy to grpc for arktos integration

### PR DESCRIPTION
**Pull request by clu2xlu**

### CODE REVIEW

File: mizar/proto/mizar/proto/builtins.proto [hunk 1]
```diff
@@ -68,6 +73,12 @@ message BuiltinsArktosMessage {
   string vpc = 2;
 }
 
+message BuiltinsNetworkPolicyMessage {
+  string name = 1;
+  string namespace = 2;
+  string tenant = 3;
```
Comment (Hong-Chang) [inline mizar/proto/mizar/proto/builtins.proto:79]: Whether network policy has tenant field?

File: mizar/arktos/arktos_service.py [hunk 2]
```diff
@@ -193,6 +211,15 @@ def DeleteService(self, request, context):
...
         return run_arktos_workflow(wffactory().k8sServiceDelete(param=param))
 
+    def DeleteNetworkPolicy(self, request, context):
+        logger.info(
+            "Deleting network policy from Network Controller {}".format(request.name))
+        param = reset_param(HandlerParam())
+        param.name = request.name
+        param.body['metadata'] = {}
+        param.body["metadata"]["namespace"] = request.namespace
+        param.extra = request.tenant
```
Comment (vinaykul) [inline mizar/arktos/arktos_service.py:221]: In create, we specify param.body['metadata']['tenant'] = request.tenant. Why is it different here?

File: mizar/proto/mizar/proto/builtins.proto (hunk 1 above)
Comment (clu2xlu) [reply mizar/proto/mizar/proto/builtins.proto:79]: Yes, it should

**Review by Hong-Chang — APPROVED** [review]
LGTM

**Pull request merged**